packages/terra-clinical-application-site/package.json

```diff
@@ -27,10 +27,9 @@
   },
   "dependencies": {
     "prop-types": "^15.5.8",
-    "react-intl": "^2.2.3",
-    "react-modal": "^1.6.5",
```

Comment:
I think it is safe to remove `react-modal` since we don't use it anymore in the demo app. We now use terra-modal-manager.